pkg/acceptance/node_test.go

```diff
@@ -47,20 +47,54 @@ const config = {
 
 const client = new pg.Client(config);
 
+function testSelect(client) {
+  return new Promise(resolve => {
+    client.query("SELECT 1 as first, 2+$1 as second, ARRAY['\"','',''] as third", [%v], function (err, results) {
+      if (err) throw err;
+
+      assert.deepEqual(results.rows, [{
+        first: 1,
+        second: 5,
+        third: ['"', '', '']
+      }]);
+
+      resolve();
+    });
+  });
+}
+
+function testSendInvalidUTF8(client) {
+  return new Promise(resolve => {
+    client.query({
+      text: 'SELECT $1::STRING',
+      values: [new Buffer([167])]
+    }, function (err, results) {
+      if (!err) throw new Error('expected error!');
```

Comment:
We should assert what the error is (similar to `testutils.IsError`)? We've run into issues in the past with wildcard error checks failing to catch behavior changes.